tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/ContextPropagation.cs

```diff
@@ -8,28 +8,33 @@
 using System.Collections.Generic;
 using System.Text;
 using Datadog.Trace.DataStreamsMonitoring;
+using Datadog.Trace.DuckTyping;
+using Datadog.Trace.ExtensionMethods;
+using Datadog.Trace.Headers;
 using Datadog.Trace.Propagators;
+using Datadog.Trace.Vendors.Newtonsoft.Json;
 
 namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AWS.SQS
 {
     internal static class ContextPropagation
     {
-        private const string SqsKey = "_datadog";
+        internal const string SqsKey = "_datadog";
 
-        private static void Inject<TMessageRequest>(SpanContext context, IDictionary messageAttributes)
+        private static void Inject<TMessageRequest>(SpanContext context, IDictionary messageAttributes, DataStreamsManager dataStreamsManager)
         {
             // Consolidate headers into one JSON object with <header_name>:<value>
             var sb = Util.StringBuilderCache.Acquire(Util.StringBuilderCache.MaxBuilderSize);
             sb.Append('{');
             SpanContextPropagator.Instance.Inject(context, sb, default(StringBuilderCarrierSetter));
+            dataStreamsManager?.InjectPathwayContext(context.PathwayContext, new StringBuilderJsonAdapter(sb));
```

Comment:
nit: anytime we touch relatively complex files like this, it's nice to add the `#nullable enable` annotations to chip away at the potential for null refs and to avoid introducing new ones 🙂 Not required for this, just a nice habit if we can get everyone to do it :D

Reply:
oh ok, I thought there was a good reason for why it was not enabled.

Reply:
> oh ok, I thought there was a good reason for why it was not enabled.

The only reason it's not enabled everywhere is because the code is older than that directive 😄  We actually have an OKR to try and increase the nullable coverage 🙂